dask/dataframe/io.py

```diff
@@ -338,8 +338,11 @@ def from_pandas(data, npartitions, sort=True):
     ----------
     df : pandas.DataFrame or pandas.Series
         The DataFrame/Series with which to construct a dask DataFrame/Series
-    npartitions : int
+    npartitions : int (defaults to None)
```

Comment:
I will change this. Note that the parenthesis format is elsewhere in the io fiie as well, namely in the `from_bcolz` function.